qiskit_experiments/data_processing/nodes.py

```diff
@@ -175,41 +187,6 @@ def _format_data(self, data: np.ndarray) -> np.ndarray:
 
         return data
 
-    @property
-    def axis(self) -> List[np.array]:
-        """Return the axis of the trained SVD"""
-        return self._main_axes
-
-    def means(self, qubit: int, iq_index: int) -> float:
-        """Return the mean by which to correct the IQ data.
-
-        Before training the SVD the mean of the training data is subtracted from the
-        training data to avoid large offsets in the data. These means can be retrieved
-        with this function.
```

Comment:
The issue with this change is that we lose the documentation. Can you add this documentation back elsewhere? Perhaps in the default options.

Reply:
Readded in https://github.com/Qiskit/qiskit-experiments/pull/639/commits/1c0b99d7f26afbe79bec1e1cb2d0d19320087dc0